frontend/src/styles/radix.css

```
@@ -98,3 +98,17 @@
+
+/* Disabling scrollbar margin management */
+html body[data-scroll-locked] {
+  /*
+    When opening a modal, Radix will add padding/margin to the body to counteract a
+    dynamic scrollbar that disappears.
+
+    Our custom scrollbar was made persistent, so we want to override this behavior,
+    otherwise the content will shift to the left anyway when opening a modal.
+  */
+  --removed-body-scroll-bar-size: 0px !important;
+  margin-right: 0px !important;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)